common/src/BasePlugin.php

```diff
@@ -36,6 +36,13 @@ public function init() {
         add_action('plugins_loaded', [$this, 'loadTranslations']);
         add_action('admin_enqueue_scripts', [$this, 'addUpdateNoticeDismissScript']);
         add_action('wp_ajax_' . $this->getUpdateNoticeDismissedAjaxHook(), [$this, 'dismissUpdateNotice']);
+        add_action( 'before_woocommerce_init', function() {
+            if ( class_exists( \Automattic\WooCommerce\Utilities\FeaturesUtil::class ) ) {
+                \Automattic\WooCommerce\Utilities\FeaturesUtil::declare_compatibility(
+                    'custom_order_tables', $this->getSlug() . '/' . $this->getSlug() . '.php', true
+                );
+            }
+        } );
```

Comment:
Please don't apply WordPress coding standards. That is, drop all the extra spaces.